btc (+testnet) 26.0 → 27.0

Dehumanizer77 opened the pull request.

martinboehm commented:
@Dehumanizer77 could you please increase `"block_addresses_to_keep": 300,` to 10000 for the testnet?

Dehumanizer77 commented:
> @Dehumanizer77 could you please increase `"block_addresses_to_keep": 300,` to 10000 for the testnet?

Changed and also updated regtest and signet which I forgot about previously.
